probe-rs/src/config/sequences/esp32.rs

```diff
@@ -60,6 +60,12 @@ impl XtensaDebugSequence for ESP32 {
         interface.write_word_32(RTC_WDTCONFIG0, 0x0)?;
         interface.write_word_32(RTC_WRITE_PROT, 0x0)?; // write protection on
 
+        tracing::warn!("");
```

Comment:
Is there a reason why these are 3 different messages, vs. one multi-line message?

In interfaces like VSCode extension, this will result in multiple independent messages, and incomplete sentences.

Reply:
It just looked better IMO. Normally I'd print this with `logging::` if it was available for the library. I didn't know warnings show up in VSCode :)

Reply:
Are you sure VSCode gets warnings? The logging is set up as:

```rust
    // We want to always log errors to stderr, but not to the log file.
    let stderr_subscriber = tracing_subscriber::fmt::layer()
        .compact()
        .with_ansi(false)
        .without_time()
        .with_writer(stderr)
        .with_filter(LevelFilter::ERROR);
```

Reply:
I meant in a more general sense. If we are changing probe-rs default logging level to WARN, then I will probably test to see if we should do the same for VSCode.